config/routes.rb

```diff
@@ -6,5 +6,10 @@
 
   get "/signin", to: "sessions#new", as: "session"
   post "/signin", to: "sessions#create"
+
   delete "/signout", to: "sessions#destroy"
+
+  namespace :admin do
+    resources :products, except: %i(show destroy)
```

Comment:
resource và resouce khác nhau ntn em

Reply:
resource nó sử dụng khi trên 1 member, nó sẽ không có id của model trong URL, còn resources thì nó sẽ hoạt động toàn bộ các resource luôn ạ